techmatters/flex-plugins
# Categories filter

Primary reviewer: @stephenhand 

## Description
This PR adds the categories filter to CaseList screen. (Front-end only)

![image](https://user-images.githubusercontent.com/1504544/166841343-eb73f834-8ae0-42a7-b40b-55922368285f.png)

![image](https://user-images.githubusercontent.com/1504544/166841385-86eb933c-4518-4933-be8e-b7811ff42167.png)

![image](https://user-images.githubusercontent.com/1504544/166841437-b855baba-dc32-4244-bdcc-dcf3d72c44c1.png)

### Checklist

### Related Issues
Jira: https://bugs.benetech.org/browse/CHI-1119

### Verification steps
Categories filter should be all functional (front-end only). You can verify the payload sent will fill the categories values.
